python/mxnet/libinfo.py

```diff
@@ -31,7 +31,7 @@ def find_lib_path():
     """
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
     api_path = os.path.join(curr_path, '../../lib/')
-    cmake_build_path = os.path.join(curr_path, '../../build/Release/')
+    cmake_build_path = os.path.join(curr_path, '../../build/')
```

Comment:
why?

Reply:
cmake artifact is put in build/ not in build/Release  Why is the previous path there I wonder...

Reply:
CMake often tends to make a separate output directory for each build configuration "Debug", "Release", "RelWithDebInfo", etc. (when tiolkd to do so, for example by CLion, except it calls it cmake-build-debug, cmake-build-release, etc.). While I am not crazy about trying to guess the cmake output directory, I think explicitly pointing to "Release" when what you think you're running is "Debug" is dangerous.

Reply:
I never saw mxnet being left in this folder, did you observe it? Maybe with a different CMake version or intellij? Agreed with your remarks.

Reply:
Someone would need to do it manually when running cmake, but it's sort of a common practice.

Reply:
I'm using Visual Studio in Windows and after building the solution file the dll will be in `build/Release`. I think this path should be added back.

Reply:
OK, forget about the previous comment. The release option is added in the codes below.